### DISCUSSION TITLE
Sprites dimensions

### DISCUSSION THREAD

**Discussion by Bur4ko** [ikemen-engine/Ikemen-GO#1725] [Feature Request] [answered]
Triggers that return the x,y dimensions of a specific sprite.

**Comment by potsmugen** [comment]
Interesting. This could be used to for instance accurately generate particle effects on the character instead of having to guesstimate where to put them. Maybe something like AnimFrame(Width) and AnimFrame(Height).

**Comment by potsmugen** [comment, marked as answer]
Implemented in `SpriteVar` trigger.
